src/open_inwoner/components/templates/components/Header/NavigationAuthenticated.html

```diff
@@ -30,6 +30,11 @@
                         </li>
                     {% endif %}
 
+                    {% if is_eherkenning_session and not is_branch_restricted %}
+                    <li class="header__list-item">
+                        {% link text=_("Wissel van vestiging") href="kvk:branches" icon="autorenew" icon_position="before" primary=True icon_outlined=True %}
+                    </li>
+                    {% endif %}
                     <li class="header__list-item">
                         {% trans "Logout" as logout %}
                         {% link text=logout href=request.user.get_logout_url icon="logout" icon_position="before" primary=True %}
```

Comment:
Just to be consistent...

```suggestion
                        {% link text=_("Logout") href=request.user.get_logout_url icon="logout" icon_position="before" primary=True %}
```